src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine excavator = new Excavator();
+        Machine truck = new Truck();
+        Bulldozer bulldozer = new Bulldozer();
+        Machine[] machines = {excavator, truck, bulldozer};
+        for (Machine m : machines) {
+            m.doWork();
+            m.stopWork();
+        }
+    }
+
```

Comment:
There is a redundant empty line here. According to the checklist: 'Please don't add redundant empty lines to your code.'